docs/src/BOILERPLATE.doc.js

```diff
@@ -0,0 +1,189 @@
+      },
+      {
+        name: 'alignItems',
+        type: `"start" | "end" | "center" | "baseline" | "stretch"`,
+        defaultValue: 'stretch',
+        description:
+          'Defines the default behaviour for how flex items are laid out along the cross-axis on the current line. Think of it as the justify-content version for the cross-axis (perpendicular to the main-axis).',
+        href: 'Flex-Layout',
+      },
+    ]}
+  />,
+);
+
+card(
+  <MainSection name="Best Practices">
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description="Description about what you should Do"
+        defaultCode={`
+<Text>
+  Code goes here
+</Text>
+`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description={`
+          What not to do goes \`here\`. Using back-tics instead of quotes allows you to use [Markdown]("https://www.markdownguide.org/")
+        `}
+        defaultCode={`
+<Text>
+  Code goes here
+</Text>
+`}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection
+    name="Accessibility"
+    description={`\`Accessibility\` guidelines go here. Can also include SubSections and Cards for detailed examples (see Box)`}
+  />,
+);
+
+card(
+  <MainSection
+    name="Localization and Inclusion"
+    description={`\`Localization and Inclusion\` guidelines go here. Can be examples in another language or truncation examples`}
+  />,
+);
+
+card(
+  <MainSection name="Variants">
+    <MainSection.Subsection description={`Description of this \`variant\`.`} title="Variant 1">
+      <MainSection.Card
+        cardSize="md"
+        title="example title"
+        defaultCode={`<Text>
+        Code goes here
+        </Text>`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        title="example title"
+        defaultCode={`<Text>
+        Code goes here
+        </Text>`}
+      />
+    </MainSection.Subsection>
+
+    <MainSection.Subsection
+      description={`To showcase repetitive examples (colors, borders, etc.), use the \`CombinationNew\` component as a child of \`MainSection.Subsection\` to render cards in the new style.`}
+      title="Variant using CombinationNew"
+    >
+      <CombinationNew
+        color={[
+          'red',
+          'white',
+          'lightGray',
+          'gray',
+          'darkGray',
+          'green',
+          'pine',
+          'olive',
+          'blue',
+          'navy',
+          'midnight',
+          'purple',
+          'orchid',
+          'eggplant',
+          'maroon',
+          'watermelon',
+          'orange',
+          'transparent',
+          'transparentDarkGray',
+          'lightWash',
+          'darkWash',
+        ]}
+      >
+        {(props) => <Box width={60} height={60} rounding="circle" {...props} />}
```

Comment:
can we avoid spreading props? 

We don't allow it (Eslint rule) in Pinboard as a best practice. Codemods can't read them.